### ISSUE TITLE
Could not convert 'Name'

### ISSUE THREAD

**Issue by ghost** [timwedde/banana-split#2]
Hey! This looks like an awesome tool. The only problem I'm having is that no matter what MIDI files I put into it I get a 'Could not convert' response for every one. I've tried a lot of different types of midi. Any idea on what I could try to get it working?

This is what it looks like 
```
banana-split.py -i ./data/midi-in -o ./data/midi-out -k -v
Converting input data...
Could not convert 'The Real Thing.2'
Could not convert 'The Real Thing.1'
Could not convert 'Tribal Dance (edit)'
Could not convert 'Maximum Overdrive'
Could not convert 'No Limit (extended)'
Could not convert 'Get Ready for This'
Cleaning input data...
Splitting channels...
Removing empty channels...
Splitting tracks...
Converting output data...
Finished processing
```

**Comment by timwedde** [comment]
Hey, I just pushed a new version with a few small tweaks.
I changed the API interface of the underlying py_midicsv library and forgot to adjust it in this project. I did that now and hopefully it works as expected. I tested it with a few MIDI files I had lying around and those worked, but of course that's not a guarantee.
In case it still fails for you, would it be possible to send me one of the failing files so that I can take a look and try to reproduce the issue?
